templates/java/TestTemplateForCLI/src/main/resources/greengrass/features/confidenceTest.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Confidence Test suite
+
+  Background:
+    Given my device is registered as a Thing
+    And my device is running Greengrass
+
+  @ConfidenceTest
+  Scenario: As a Developer, I can deploy GDK_COMPONENT_NAME, stop it and start it again
+    When I create a Greengrass deployment with components
+      | GDK_COMPONENT_NAME | GDK_COMPONENT_RECIPE_FILE |
+      | aws.greengrass.Cli        | LATEST |
+    And I deploy the Greengrass deployment configuration
+    Then the Greengrass deployment is COMPLETED on the device after 180 seconds
+    # Update component state accordingly. Possible states: {RUNNING, FINISHED, BROKEN, STOPPING}
+    Then I verify the ${GDK_COMPONENT_NAME} component is RUNNING using the greengrass-cli
```

Comment:
This has to be `GDK_COMPONENT_NAME` only. `gdk test-e2e build` will only replace the `GDK_COMPONENT_NAME` string  so `${GDK_COMPONENT_NAME}` still going to be something like `$com.aws.HelloWorld`.